PaymentInitiation/Program.cs

```diff
@@ -52,23 +52,38 @@ public class Settings
         public class Payment
         {
             public string BicFi { get; }
+            public string AffiliatedASPSPId { get; }
+            public string PSUId { get; }
+            public string PSUCorporateId { get; }
             public string PaymentService { get; }
             public string PaymentProduct { get; }
+            public string AuthenticationMethodId { get; }
             public string PaymentBody { get; }
             public string PaymentId { get; set; }
             public string PaymentAuthId { get; set; }
             public SCAMethod ScaMethod { get; set; }
-            public string ScaData { get; set; }
+            public SCAData ScaData { get; set; }
 
-            public Payment(string bicFi, string paymentService, string paymentProduct, string paymentBody)
+            public Payment(string bicFi, string affiliatedASPSPId, string psuId, string psuCorporateId, string paymentService, string paymentProduct, string authenticationMethodId, string paymentBody)

```

Comment:
Some constructors are now taking quite a few overloads, is this to be clear on what is needed, or could it be replaced by sending for example the settings object and populate properties from there?